dev-workflow-ui/webContent/includes/template/notifications.xhtml

```diff
@@ -0,0 +1,63 @@
+<ui:composition xmlns="http://www.w3.org/1999/xhtml" xmlns:f="http://xmlns.jcp.org/jsf/core"
+  xmlns:h="http://xmlns.jcp.org/jsf/html" xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+  xmlns:c="http://xmlns.jcp.org/jsp/jstl/core" xmlns:ic="http://ivyteam.ch/jsf/component"
+  xmlns:p="http://primefaces.org/ui" xmlns:pe="http://primefaces.org/ui/extensions">
+
+<p:commandLink id="showNotifications">
+  <i class="topbar-icon pi pi-bell"/>
+  <div class="notification-badge">
+    <h:outputText id="unreadNotifications" value="#{notificationBean.unreadNotifications}" />
+  </div>
+</p:commandLink>
+
+<p:overlayPanel for="showNotifications" style="width:350px">
+  <h:panelGroup id="notifications">
+    <div class="p-d-flex p-jc-between">
+      <h6>Notifications</h6>
+
+      <div class="p-d-flex">
+        <p:commandLink id="notificationMarkAllAsRead" rendered="#{notificationBean.hasUnreadNotifications()}" actionListener="#{notificationBean.readAll()}" update="notifications unreadNotifications">
+          <i class="notification-button si si-check-circle-1 p-mr-1" />
+        </p:commandLink>
+        <p:tooltip for="notificationMarkAllAsRead" value="Mark all notifications as read" />
+
+        <p:commandLink id="notificationDeleteAll" rendered="#{notificationBean.hasNotifications()}" actionListener="#{notificationBean.deleteAll()}" update="notifications unreadNotifications">
+          <i class="notification-button si si-delete-1" />
+        </p:commandLink>
+        <p:tooltip for="notificationDeleteAll" value="Delete all notifications" />
+      </div>
+    </div>
+
+    <h:panelGroup rendered="#{!notificationBean.hasNotifications()}">
+      <div class="p-text-center" style="padding:20px;">No Notifications </div>
+    </h:panelGroup>
+
+    <h:panelGroup rendered="#{notificationBean.hasNotifications()}">
+      <div id="notification-scroll">
+
+        <p:repeat value="#{notificationBean.notifications}" var="notification">
+          <div class="#{notification.style} notification-container">
+            <div class="p-d-flex p-jc-between">
+              <h:outputText value="#{notification.message}" class=""></h:outputText>
+
+              <div class="p-d-flex">
+                <p:commandLink id="notificationMarkAsRead" rendered="#{!notification.read}" actionListener="#{notificationBean.markAsRead(notification)}" update="notifications unreadNotifications">
+                  <i class="notification-button si si-check-circle-1 p-mr-1 " />
+                </p:commandLink>
+                <p:tooltip for="notificationMarkAsRead" value="Mark as read" />
+              </div>
+            </div>
+
+            <div>
+              <h:outputText value="#{notification.createdAt}" style="font-size: 0.75em;">
+                <f:convertDateTime type="both" dateStyle="medium" timeStyle="short" />
+              </h:outputText>
+            </div>
+          </div>
```

Comment:
I would add here the typical "unread" dot
![image](https://github.com/axonivy/dev-workflow-ui/assets/45038833/68ec21d5-bf92-4c1f-84eb-49589718b50a)
https://www.primefaces.org/freya/dashboard.xhtml#

Reply:
I would also add some type of separation between messages. either make their container background different color, or border etc.
![image](https://github.com/axonivy/dev-workflow-ui/assets/45038833/0da51cbb-ccf7-4e64-8bda-287305163233)
https://www.primefaces.org/freya/dashboard.xhtml#